zrml/prediction-markets/src/lib.rs

```diff
@@ -333,6 +332,10 @@ mod pallet {
             let market_status = market.status;
             let market_account = <zrml_market_commons::Pallet<T>>::market_account(market_id);
 
+            if Self::is_dispute_bond_pending(&market_id, &market, false) {
+                Self::unreserve_dispute_bond(&market_id)?;
+            }
+
```

Comment:
so when a market is destroyed we should slash the dispute bond instead of unreserving it. what is your argument for this? idk why i did it this way (unreserve). i think you told me to unreserve it... but not sure.

Reply:
https://github.com/zeitgeistpm/zeitgeist/commit/5d6df7961b83d185a26b3e2b69a2f1f4b9005eb3